build/pkgs/jupymake/distros/conda.txt

```diff
@@ -0,0 +1 @@
+# The presence of this file ensures that we do not try to pip-install JuPyMake (there is no polymake conda package).
```

Comment:
Need to do the same for pycygwin

Reply:
I think the pip install of pycygwin is just a no-op

Reply:
(on platforms other than cygwin)

Reply:
It fails with
```
[1/1] Cythonizing cygwin/_cygwin.pyx
        /usr/share/miniconda/envs/sage-build/lib/python3.9/site-packages/Cython/Compiler/Main.py:369: FutureWarning: Cython directive 'language_level' not set, using 2 for now (Py2). This will change in a later release! File: /tmp/pip-install-zc0098ck/pycygwin_2ac898eefcb940e7b7386efa9d3af878/cygwin/_cygwin.pxd
          tree = Parsing.p_module(s, pxd, full_module_name)
        build/cygwin/_cygwin.c:759:10: fatal error: sys/cygwin.h: No such file or directory
          759 | #include <sys/cygwin.h>
              |          ^~~~~~~~~~~~~~
        compilation terminated.
        error: Command "/usr/share/miniconda/envs/sage-build/bin/x86_64-conda-linux-gnu-cc -Wno-unused-result -Wsign-compare -DNDEBUG -fwrapv -O2 -Wall -fPIC -O2 -isystem /usr/share/miniconda/envs/sage-build/include -fPIC -O2 -isystem /usr/share/miniconda/envs/sage-build/include -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -isystem /usr/share/miniconda/envs/sage-build/include -DNDEBUG -D_FORTIFY_SOURCE=2 -O2 -isystem /usr/share/miniconda/envs/sage-build/include -fPIC -I/usr/share/miniconda/envs/sage-build/include/python3.9 -c build/cygwin/_cygwin.c -o build/temp.linux-x86_64-cpython-39/build/cygwin/_cygwin.o" failed with exit status 1
```
on CI

Reply:
Sorry, obviously you're right!

Reply:
Done in a3772d55ce